samples/estimation/df-chemistry/chemistry.py

```diff
@@ -294,7 +294,7 @@ def eigen_decomp(
         matrix[np.tril_indices(dimension, 0)] = vector
 
         # Compute the eigen decomposition of the lower triangular matrix
-        evals, evecs = LA.eigh(matrix, lower=True)
+        evals, evecs = LA.eigh(matrix, UPLO="L")
```

Comment:
```suggestion
        evals, evecs = np.linalg.eigh(matrix, UPLO="L")
```

to be consistent with the following two lines. In that case the new import can also be removed.

Reply:
I fixed them all to use LA consistently, as otherwise the numpy.linalg submodule isn't actually imported, which in Python it is supposed to  be before it is used.